src/aleph/schemas/api/accounts.py

```diff
@@ -69,10 +74,31 @@ class AddressBalanceResponse(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
     address: str
-    balance: str
+    balance: FloatDecimal
```

Comment:
Why changing this to float? We put as a string because the token balance can have more decimals than the allowed overflow for float values.

Reply:
This model is the response schema for the endpoint `https://api.aleph.im/api/v0/balances` which is not being use anywhere. It is currently broken in production nodes, becase the type casting from Decimal to str is not working.

I have fixed and aligned this schema to have the same format than `/api/v0/addresses/{address}/balance` which is  `FloatDecimal => float`
https://github.com/aleph-im/pyaleph/blob/main/src/aleph/schemas/api/accounts.py#L24